wrappers/rust/cpp-client-telemetry-sys/src/lib.rs

```diff
@@ -0,0 +1,90 @@
+#![allow(non_upper_case_globals)]
+#![allow(non_camel_case_types)]
+#![allow(non_snake_case)]
+include!(concat!(env!("OUT_DIR"), "/bindings.rs"));
+
+use std::ffi::*;
+
+fn evt_api_call_wrapper(evt_context: Box<evt_context_t>) -> (evt_status_t, Box<evt_context_t>) {
```

Comment:
Why does this take ownership of the `Box<evt_context_t>` and then return it?  My naive reading of this would be that `evt_context: &evt_context_t` would be the right type.